src/main/java/org/gridsuite/actions/server/dto/FormContingencyList.java

```diff
@@ -30,15 +30,18 @@ public class FormContingencyList implements ContingencyList {
     @Schema(description = "Equipment type")
     private String equipmentType;
 
-    @Schema(description = "Nominal voltage")
-    private double nominalVoltage;
+    @Schema(description = "Nominal voltage 1")
+    private NumericalFilter nominalVoltage1;
 
-    @Schema(description = "Nominal voltage operator")
-    private String nominalVoltageOperator;
+    @Schema(description = "Nominal voltage 2")
+    private NumericalFilter nominalVoltage2;
 
     @Schema(description = "Countries")
     private Set<String> countries;
 
+    @Schema(description = "Countries")
+    private Set<String> countries2;
```

Comment:
use same naming as nominalVoltage: countries1 and countries2

Reply:
yes, it was a TODO for me at the beginning, but I finally kept the original name.
I will rename.